index.js

```diff
@@ -0,0 +1,82 @@
+//insert, connect required modules
+//for terminal dialogue
+const inquirer = require("inquirer");
```

Comment:
nice job importing all your packages and files here - this is great for readability, particularly on larger projects :-)